### DISCUSSION TITLE
How about `ble-face -s auto_complete fg=8` by default to keep same behaviour with https://github.com/zsh-users/zsh-autosuggestions?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [akinomyoga/ble.sh#463] [Ideas]
How about `ble-face -s auto_complete fg=8` by default to keep same behaviour with https://github.com/zsh-users/zsh-autosuggestions?

**Comment by akinomyoga** [comment]
There are reasons for the current default.

- First, the line editor has no way to robustly know the terminal's background color, in particular, whether it's a light background or a dark background. For this reason, I would like to keep the face work in both cases of light and dark background colors.
- Second, `ble.sh` also provides syntax highlighting for many different contexts (unlike zsh-autosuggestions which just provides the auto-suggestion). I would like to assign a distinct graphic style to each type of syntactic/contextual element. The gray foreground color is already used for the comments in Bash syntax.
 
Nevertheless, I understand that users may want to use different graphic style for the autosuggestions, so `ble.sh` provides the way to configure it. Is there a problem with the current configuration interface?

**Reply by Freed-Wu** [reply]
No problem. Thanks for your answer :+1:
